internal/app/machined/pkg/controllers/block/volume_config.go

```diff
@@ -124,6 +124,8 @@ func convertEncryptionConfiguration(in cfg.EncryptionConfig, out *block.VolumeCo
 		case key.TPM() != nil:
 			out.Encryption.Keys[i].Type = block.EncryptionKeyTPM
 			out.Encryption.Keys[i].TPMCheckSecurebootStatusOnEnroll = key.TPM().CheckSecurebootOnEnroll()
+			out.Encryption.Keys[i].TPMPCRs = key.TPM().PCRs()
+			out.Encryption.Keys[i].PubKeyPCRs = []int{constants.UKIPCR}
```

Comment:
it seems that we don't consume this value anywhere?

Reply:
I mean in the tpm2 code, we don't use this value, but rather use the constant once again - my point is - should we actually use value from the resource? (I'm all for exposing this in a resource)

Reply:
fixed